### ISSUE TITLE
Typeahead suggestions should be readable by a screen reader

### ISSUE THREAD

**Issue by nickmarden** [GSA/search-gov#33]
Reported by a customer: screen readers are not able to read or use the typeahead search suggestions in their dropdown list as we present it via the javascript. The USA.gov team reviewed it, and proposed the following fix. (They warned us that some assistive tech scanners would continue to flag it as an issue, but it's the only thing they could identify that would let screen reader access the options.)

From David Stenger:
https://haltersweb.github.io/Accessibility/autocomplete.html This example is how it should be coded and how to navigate the field with AT.

**Comment by MothOnMars** [comment]
Waiting for confirmation that this issue is valid.

**Comment by dawnpm** [comment]
Resolved by https://github.com/GSA/search-gov/commit/65460992a0135886c32ce61ca77e24074c4b38c8
